app/controllers/concerns/statistics.rb

```diff
@@ -0,0 +1,32 @@
+module Statistics
+  extend ActiveSupport::Concern
+
+  def dates_and_ranges
+    @start = DateTime.parse('2017-01-01').in_time_zone
+    @one_month = 1.month.ago..Time.zone.now
+    @one_year = 1.year.ago..Time.zone.now
+    @from_start = @start..Time.zone.now
+  end
+
+  def period
+    dates_and_ranges
```

Comment:
what is dates_and_ranges ? rename method to something so I could understand what it's doing